Documentation/Signoffs/SteeringSystem.md

```diff
@@ -86,30 +49,46 @@ where C is the number of counts, CPR is the Counts per Revolution, and A is the
 
 Similarly, the accuracy of the angle is highly important to the selection of an encoder. To find the output sensitivy of the encoder, the following equation is used:
      
-     D = 360/(Resolution * 2)       (2)
+     D = 360/(Resolution)       (2)
 
 where D is the degrees of accuracy and Resolution is the how many data bits. 
 
-##### 4. Encoder Selection
+##### 2. Encoder Selection
+
+This steering wheel design will use a TRD-NA1024NW absolute rotary encoder with a resolution of 1024. The Encoder a voltage rating of 12 to 24 VDC, and a maximum current consumption of 70 mA. The TRD-NA1024NW has 10 data bits, a VDD pin, and a GND pin. This requires a microcontroller with at least 10 digital GPIO pins and that can supply the current of up to 70 mA to the rotary encoder.
 
-This steering wheel design will use a TRD-NA1024NW absolute rotary encoder with a resolution of 1024. The Encoder has a max frequency of 20 kHz, a voltage rating of 12 to 24 VDC, and a maximum current consumption of 70 mA. Given the maximum frequency of 20 kHz, the sampling rate must be at least 40 kHz to satify the Nyquist Theorem. Using equation 2 above, the degrees of accuracy for this encoder is 0.176 degrees which means it is capable of measuring the angle of the steering wheel up to 20% of a degree. The TRD-NA1024NW has 10 data bits, a VDD pin, and a GND pin. This requires a microcontroller with at least 10 digital GPIO pins and that can supply the current of up to 70 mA to the rotary encoder.
+The encoder has 1024 unique 10-bit outputs than will be read into the MCU. The output of the encoder changes on rotation of the knob. The knob of the encoder will be connected to a gear system that is mounted to the steering column of the vehicle so that the steering wheel rotation will rotate the knob. Each 10-bit output will increment or decrement a count variable in the MCU code depending on the driection of rotation. That count will be used to calculate the angle of the wheel using equation 1 above.
 
-##### 5. Microcontroller Selection
+##### 3. Microcontroller Selection
 
 The Arduino UNO REV3 is clocked at 16 MHz, has 14 digital I/O pins, and 6 analog I/O pins. The board can be powered via battery, USB, or the VDD pin and can operate within a voltage range of 7-12 V. Each GPIO pin can supply up to 40 mA, but normally operates at 20 mA per pin. Given that the encoder requires 10 digital pins, the Arduino UNO can support that requirement and still have leftover pins for communication. Addtionally, the power supplied to the encoder will be supplied by the power system to avoid any possible damage to the microcontroller.
```

Comment:
This does not sound like all the electrical considerations have been taken. Why is the current supplied by the pins of concern? The encoder will output to the microcontroller, not the other way around. Further, what about voltage?

Reply:
You are correct, I had it backwards when I first wrote it. I fixed it in the document. The Arduino pins can receive up to 40 mA and operate at 5 V. The Encoder outputs a maximum of 32 mA and the pins output 10-bit greycode.